grind/bin/grind

```diff
@@ -355,6 +355,10 @@ class TestRunner:
                             type=int,
                             default=default_timeout,
                             help="Per-task timeout. The default timeout is %s seconds." % default_timeout)
+        parser.add_argument('--java-version',
+                            type=int,
+                            default=7,
```

Comment:
Default 7 gives users backwards compatibility, however I am open to 8 as default.